x/data/server/testsuite/suite.go

```diff
@@ -41,106 +42,168 @@ func (s *IntegrationTestSuite) TearDownSuite() {
 	s.fixture.Teardown()
 }
 
-func (s *IntegrationTestSuite) TestScenario() {
-	//testContent := []byte("xyzabc123")
-	//mh, err := multihash.Sum(testContent, multihash.SHA2_256, -1)
-	//s.Require().NoError(err)
-	//cid := gocid.NewCidV1(gocid.Raw, mh)
-	//
+func (s *IntegrationTestSuite) TestGraphScenario() {
+	testContent := []byte("xyzabc123")
+	hash := crypto.BLAKE2b_256.New()
+	_, err := hash.Write(testContent)
+	s.Require().NoError(err)
+	digest := hash.Sum(nil)
+	graphHash := &data.ContentHash_Graph{
+		Hash:                      digest,
+		DigestAlgorithm:           data.DigestAlgorithm_DIGEST_ALGORITHM_BLAKE2B_256,
+		CanonicalizationAlgorithm: data.GraphCanonicalizationAlgorithm_GRAPH_CANONICALIZATION_ALGORITHM_URDNA2015,
+	}
+	contentHash := &data.ContentHash{Sum: &data.ContentHash_Graph_{Graph: graphHash}}
+
+	//// anchor some data
+	anchorRes, err := s.msgClient.AnchorData(s.ctx, &data.MsgAnchorDataRequest{
+		Sender: s.addr1.String(),
+		Hash:   contentHash,
+	})
+	s.Require().NoError(err)
+	s.Require().NotNil(anchorRes)
+
+	// anchoring same data twice is a no-op
+	_, err = s.msgClient.AnchorData(s.ctx, &data.MsgAnchorDataRequest{
+		Sender: s.addr1.String(),
+		Hash:   contentHash,
+	})
+	s.Require().NoError(err)
+
+	// can query data and get timestamp
+	queryRes, err := s.queryClient.ByHash(s.ctx, &data.QueryByHashRequest{
+		Hash: contentHash,
+	})
+	s.Require().NoError(err)
+	s.Require().NotNil(queryRes)
+	s.Require().NotNil(queryRes.Entry)
+	ts := queryRes.Entry.Timestamp
```

Comment:
It would be nice in this test to actually verify the timestamp is accurate (especially as we are calling `AnchorData()` multiple times and asserting the second is a no-op.